test/pages/product/index.spec.tsx

```diff
@@ -1,18 +1,18 @@
 import { useRouter } from "next/router";
-import SidePanel from "@pages/product/[productId]"; 
+import ProductAppExtension from "@pages/productAppExtension/[productId]"; 
 import { render, screen} from '@test/utils';
 
 jest.mock('@lib/hooks', () => require('@mocks/hooks'));
 jest.mock('next/router', () => ({
     useRouter: jest.fn(),
 }));
 
-describe('SidePanel', () => {
+describe('ProductAppExtension', () => {
     const router = { query: { pid: '1' } };
     useRouter.mockReturnValue(router);
 
     test('renders correctly', async () => {
```

Comment:
we have an opening `async`, but no `await` in this test

Reply:
updated